api-controllers/ServiceBrokerApiController.js

```diff
@@ -101,6 +101,10 @@ class ServiceBrokerApiController extends FabrikBaseController {
         metadata: {
           finalizers: [`${CONST.APISERVER.FINALIZERS.BROKER}`]
         },
+        labels: {
+          organizationGuid: _.get(params, 'organization_guid'),
```

Comment:
Let us make it generic. We put everything that is there in the context. Just loop through context ad put everything. That should also include 'organization_guid'

Reply:
Done.